### DISCUSSION TITLE
Are faster calls to focus events possible?

### DISCUSSION THREAD

**Discussion by kyle-ssg** [software-mansion/react-native-screens#1334] [Ideas]
If I use a custom navigation bar, calls to pop immediately trigger focus events, however native gestures/clicking native back buttons have a significant delay in calling these.

Is it possible to dispatch these events earlier?

**Comment by tboba** [comment]
Hi @kyle-ssg, sorry for the late response 🙏 
can you reproduce your scenario of application which sends focus events too late? A video representing what are your intentions would be helpful there.
